### ISSUE TITLE
Enhancement: visible icon hover states in gallery lightbox 

### ISSUE THREAD

**Issue by ilonagl** [Automattic/jetpack#27770]
### Impacted plugin

None / Other

### What

Make sure the background changes when hovered over the icons

### How

Issue: currently hovering over icons (as marked in the first image) has no effect to the background and thus are barely visible.

Task: making sure the hover state changes the background around the icons (from black to #7F7F7F).

Note: it seems that the background changes when the state is active (as shown in the second image) but nevertheless it's not visible during the hover. The same background size & paddings should be reused.

![Hover states](https://user-images.githubusercontent.com/100145243/205936806-80441692-1f49-433c-9f04-bedf27754513.png)
<img width="144" alt="Screenshot 2022-12-06 at 16 27 00" src="https://user-images.githubusercontent.com/100145243/205938495-6b788c78-0bdf-4a11-a051-5ef99a866a02.png">

**Comment by jeherve** [comment]
Somewhat related: #25326
